src/neo/SmartContract/StackItemSerializer.cs

```diff
@@ -177,6 +177,8 @@ private static void Serialize(StackItem item, BinaryWriter writer)
                         writer.Write((byte)StackItemType.Null);
                         break;
                 }
+                if (writer.BaseStream.Position > maxSize)
```

Comment:
I prefer to increment a counter, maybe this BaseStream is not `seekable`

Reply:
This is a private method and the `BaseStream` is a `MemoryStream`.